docs/json/samples/event-payload-onHand-sample.json

```diff
@@ -0,0 +1,12 @@
+{
+    "id": "div1:store005:0003400029005",
+    "divisionId": "div1",
+    "storeId": "store005",
+    "upc": "0003400029005",
+    "countAdjustment": 1000,
+    "type": "onhandUpdate",
```

Comment:
We just changed the event type to be camel case uniform with the rest of the project so it should now read
`"type": "onHandUpdate"`

Reply:
Ok I will update that.

Reply:
Done